builtin/providers/aws/resource_aws_route53_record_test.go

```diff
@@ -104,6 +104,9 @@ func testAccCheckRoute53RecordExists(n string) resource.TestCheckFunc {
 }
 
 const testAccRoute53RecordConfig = `
+provider "aws" {
+    region = "us-east-1"
+}
```

Comment:
Worth noting that other acceptance tests assume `us-west-2` in their selection of AMI IDs and the like. I've been meaning to enforce this at the top level in the acceptance test suite since I discovered it.